ArduCopter/version.h

```diff
@@ -1,12 +1,15 @@
 #pragma once
 
-#include "ap_version.h"
+#include "AP_Common/AP_FWVersion.h"
 
 #define THISFIRMWARE "APM:Copter V3.6-dev"
+
+// the following line is parsed by the autotest scripts
 #define FIRMWARE_VERSION 3,6,0,FIRMWARE_VERSION_TYPE_DEV
```

Comment:
> As mentioned in the comment - that line is parsed by the autotest scripts!

Sorry :disappointed:

> The autotest scripts will not to macro expansion :-)

What? It should! 😛 

I'm not sure vehicle maintainers will like very much to have three places to change now instead of the current two, but other than that, this specific change is fine to me. My only complaints about this PR remain in previous comments.